subsys/testsuite/ztest/include/zephyr/ztress.h

```diff
@@ -159,10 +159,14 @@ struct ztress_context_data {
 #define ZTRESS_EXECUTE(...) do {							\
 	Z_ZTRESS_TIMER_CONTEXT_VALIDATE(__VA_ARGS__);					\
 	int has_timer = Z_ZTRESS_HAS_TIMER(__VA_ARGS__);				\
-	struct ztress_context_data data[] = {						\
+	struct ztress_context_data data1[] = {						\
```

Comment:
I'm not sure I get why this indirection - why not make `data` static directly?

Reply:
Uh... it's indeed too ugly. I will just add the `static`. Please see my latest push.